server/api.go

```diff
@@ -298,13 +298,11 @@ func (p *Plugin) handleConfig(w http.ResponseWriter) error {
 
 	type config struct {
 		clientConfig
-		SkuShortName         string `json:"sku_short_name"`
-		CloudMaxParticipants int    `json:"cloud_max_participants"`
+		SkuShortName string `json:"sku_short_name"`
 	}
 	ret := config{
-		clientConfig:         p.getConfiguration().getClientConfig(),
-		SkuShortName:         skuShortName,
-		CloudMaxParticipants: cloudMaxParticipants,
```

Comment:
I realize this is a breaking change but given it only affects Cloud I think we can *safely* make it now.